libmodman/test/main.cpp:35:16: error: invalid conversion from
‘int (__attribute__((__stdcall__)) *)(SOCKET, char*, int, int)
{aka int (__attribute__((__stdcall__)) *)(unsigned int, char*, int, int)}’
to ‘void*’ [-fpermissive]

--- libproxy-0.4.11/libmodman/test/main.cpp	2010-07-29 07:14:59.000000000 -0500
+++ libproxy-0.4.11/libmodman/test/main.cpp	2013-11-20 00:36:40.128436500 -0600
@@ -32,7 +32,7 @@ int main(int argc, const char** argv) {
 	module_manager mm;
 #ifdef WIN32
 #ifdef SYMB
-	void* symb = recv;
+	void* symb = (void*)recv;
 #endif
 #endif
 
